modules/administration/pages/kubernetes.adoc

```diff
@@ -126,25 +122,28 @@ The following steps will guide you through importing a previously deployed image
 
 The following steps will help you find additional runtime data.
 
-. Select to menu:Images[Images] , click the `Details` button on the right end of a row which has running instances.
-. Under the `Overview` tab, notice the data in `Runtime` and `Instances` fields under `Image Info` section.
+. Select menu:Images[Image List].
+. Click the `Details` button on the right end of a row that has running instances.
+. Under the `Overview` tab, notice the data in `Runtime` and `Instances` fields under the `Image Info` section.
 . Select the `Runtime` tab.
-. Here is a breakdown of the Kubernetes pods running this image in all the registered clusters including the following data:
+. In the `Runtime` tab is a breakdown of the Kubernetes pods running this image in all the registered clusters including the following data:
++
 
 ** Pod name
 ** Namespace which the pod resides in
 ** The runtime status of the container in the specific pod.
-Status icons are explained in the preceeding example.
+Status icons are explained in the next section.
 
 
 
 [[at.rebuild.a.previously.deployed.image.in.kubernetes]]
 == Rebuild a Previously Deployed Image in Kubernetes
 
 
-The following steps will guide you through rebuilding an image which has been deployed to a Kubernetes cluster.
+The following steps will guide you through rebuilding an image that has been deployed to a Kubernetes cluster.
 
-. Go to menu:Images[Images] , click the Details button on the right end of a row which has running instances.
+. Go to menu:Images[Image List].
```

Comment:
These two steps should be together (same as L125)

Reply:
ok